lib/sageone_sdk/default.rb

```diff
@@ -1,8 +1,9 @@
 require 'sageone_sdk/version'
 
 module SageoneSdk
+  # Default
   module Default
-    ACCESS_TOKEN = "default_access_token"
+    ACCESS_TOKEN = "ACCESS_TOKEN"
```

Comment:
Ah, we probably don't need to make that change now we figured out the failing spec issue.